website/source/docs/agent/telemetry.html.md

```diff
@@ -48,11 +48,84 @@ Below is sample output of a telemetry dump:
 
 # Key Metrics
 
```

Comment:
Can we just add a quick line here:

> These are some metrics emitted that can help you understand the health of your cluster at a glance. For a full list of metrics emitted by Consul, see <below>

Where below links to the table of all metrics.